docs/en/install-upgrade/release-notes/release-notes-fleet-agent.asciidoc

```diff
@@ -0,0 +1,147 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes-fleet-agent-9.0.0]]
+= {fleet} and {agent} Release notes
+++++
+<titleabbrev>{fleet} and {agent}</titleabbrev>
+++++
+
+This section summarizes the changes in each release.
+
+* <<release-notes-fleet-agent-9.0.0-beta1>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
```

Comment:
> * {kibana-ref}/release-notes.html[{kib} release notes]

These URLs should point to the release notes in this guide rather than in the kibana and beats guides. I will fix these links after https://github.com/elastic/stack-docs/pull/2970/ is merged (since it changes the URLs).

Reply:
Thanks @lcawl I'll leave this PR in your super capable hands. :-)

Reply:
The other PR keeps hitting broken links, so I'll merge this one first and circle back to improve the RN links thereafter.